src/clusterfuzz/_internal/bot/tasks/cron/libs/request_cache.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Google LLC
```

Comment:
This isn't usable outside of a web server context. Let's skip/remove this completely for the new cron.